functions/src/modules/SE.functions.ts

```diff
@@ -0,0 +1,78 @@
+import * as functions from 'firebase-functions';
+import * as  admin from 'firebase-admin';
+
+import uniqid from 'uniqid';
+
+const db = admin.database();
+
+/////////////////////////////////////////////////
+//          updateNotifyOnNewAllotment (DB create Trigger)
+//      1. Mark the task/s in the database as given --> { status: "Given" }
+//      1. Set the task/s `assignee`, and `timestampGiven`
+//
+//      2. Send an email to the assignee to notify them of the new allotment
+//
+/////////////////////////////////////////////////
+export const updateNotifyOnNewAllotment = functions.database
+.ref('/sound-editing/restoration/allotments/{pushId}').onCreate(async (snapshot, context) => {
+    const coordinatorConfig = functions.config().coordinator;
+
+    const allotment = snapshot.val();
+
+    let timestamp = new Date(allotment['timestamp']);
+
+    //  1. Ensure sound editor's `uploadCode`
+    let userRef = await db.ref('/users').orderByChild('emailAddress')
+            .equalTo(allotment.assignee.emailAddress).once('value');
+    if (!userRef.exists())
+        return -1;
+    let user = userRef.val();
+    let { uploadCode } = user;
+    if (!uploadCode) {
+        uploadCode = uniqid();
+        db.ref('/users').orderByChild('emailAddress')
+            .equalTo(allotment.assignee.emailAddress).ref.child('uploadCode').set({ uploadCode });
+    }
+
+    // 2. Update the task
+    const { taskIds } = allotment;
+    const tasks = [];
+    taskIds.forEach(async taskId => {
+        const regex = /(\w+)-(\d+)-(\d+)/g;
+        let taskIdMatch = regex.exec(taskId);
+        const list = taskIdMatch[1];
+
+        await db.ref(`/sound-editing/tasks/${list}/${taskId}/restoration`).update({
+            status: 'Given',
+            assignee: allotment.assignee,
+            timestampGiven:Math.round((new Date()).getTime() / 1000),
```

Comment:
Do you have any linter enabled? It should warn and even fix such cases (here missing a space after a colon). Please configure your VS Code to highlight and fix the linter errors on save.